WinUIGallery/Samples/ControlPages/Fundamentals/Controls/CounterControl.cs

```diff
@@ -0,0 +1,77 @@
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+namespace WinUIGallery.Samples.ControlPages.Fundamentals.Controls;
+
+public enum CounterMode
+{
+    Increment,
+    Decrement
+}
+
+public sealed class CounterControl : Control
+{
+    public static readonly DependencyProperty CountProperty =
+        DependencyProperty.Register(nameof(Count), typeof(int), typeof(CounterControl), new PropertyMetadata(0));
+
+    public static readonly DependencyProperty ModeProperty =
+        DependencyProperty.Register(nameof(Mode), typeof(CounterMode), typeof(CounterControl), new PropertyMetadata(CounterMode.Increment));
```

Comment:
Would it be helpful to show how you could trigger an event whenever this value changes?